assembly/assembly-main/src/assembly/bin/che.sh

```diff
@@ -663,77 +360,25 @@ launch_docker_registry () {
     fi
 
     if ${CREATE_NEW_CONTAINER} ; then
-
       # Container in bad state or not found, kill and launch new container.
       kill_and_launch_docker_registry
-
     fi
 }
 
-launch_che_server () {
-
-  if [[ -n "${DOCKER_HOST}" ]]; then
-    strip_url ${DOCKER_HOST}
-  else
-    host=localhost
-  fi
-
-  if ! ${SKIP_PRINT_CLIENT}; then
-    print_client_connect
-  fi
-
-  if ${LAUNCH_REGISTRY} ; then
-    # Export the value of host here
-    # Will be used on Che properties to set the location of registry
-    export CHE_REGISTRY_HOST="${host}"
-    launch_docker_registry
+execute_che () { 
+  check_docker
+  determine_os
```

Comment:
We will have to manage that in another PR, I believe.  The scope of the current scirpt is related to the assumption that you are running the container on a host without privileged mode and /var/run/docker.sock activated.

We have two scenarios that we have to address with the che-server container, and the underlying che.sh Tomcat script:
1: - if it is a non-privileged container, then we need to rewrite certain functions - or pass in environment values from the container orchestrator.
2: - we have customers that are going to require Che in localhost mode - without the use of any Docker capabilities.  This will need to come a bit later this year.  But we make a large number of assumptions about Docker in the current scripts which would not be necessary and must be removed as well.